### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by varun-Tad**
Created Card components

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -301,16 +302,159 @@ <h2 class="component-sub-heading">Examples of Badges</h2>
...
+                  <h4 class="card-subtitle"><small>by James Clear</small></h4>
+                  <p class="card-text description-container">This breakthrough book from James Clear is the most comprehensive guide on how to change your habits and get 1% better every day.</p>
+               </div>
+           </div>
+           <div class="link-container">
+             <button class="btn btn-secondary">Buy now</button>
+             <button class="btn btn-secondary">Bookmark</button>
+             <i class="far fa-heart fa-1x"></i>
+             <i class="fas fa-share-alt fa-1x"></i>
+             <i class="fas fa-ellipsis-v fa-1x"></i>
```
Comment (rdthetha) [inline documentation.html:447]: The icon could be made responsive!

File: documentation.html [hunk 2]
```diff
@@ -301,16 +302,159 @@ <h2 class="component-sub-heading">Examples of Badges</h2>
...
+        style="width: 1024px; height: 614px; border:0; transform: scale(1); overflow:hidden;"
+        sandbox="allow-scripts allow-same-origin">
```
Comment (rdthetha) [inline documentation.html]: The inline styling for carbon could me moved to css file

File: Components/Cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+.image-container{
+    width:370px;
+}
+
+.image{
+    width:100%;
+    border-top-left-radius: 1.5rem;
+    border-top-right-radius: 1.5rem;
+}
```
Comment (Naman-Saxena1) [inline Components/Cards/cards.css:98]: To reduce code, you can use border-radius shorthand syntax.

File: Components/Cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+.text-container{
+    margin-left:20px;
+    margin-top:10px;
+    
+}
+
+.card-subtitle{
+    margin-left:25px;
+    margin-top:5px;
```
Comment (Naman-Saxena1) [inline Components/Cards/cards.css:109]: Same, you can use margin shorthand, they are pretty handy.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
I suggest you look into margin and border-radius shorthand. Also as per LMS PRD you forgot to make "Card with Dismiss".

**Comment by varun-Tad** [comment]
Thanks for reminding me about Cards with Dismiss. I missed it. Also yes I'll look into the shorthand too and implement it. Thanks for the review 👍 @Naman-Saxena1

File: documentation.html (hunk 2 above)
Comment (varun-Tad) [reply documentation.html]: Okay, will work on it 👍 .Thanks for the review @rdthetha

**Pull request merged**